keras_retinanet/bin/train.py

```diff
@@ -17,6 +17,7 @@
 """
 
 import argparse
+import functools
```

Comment:
Hmm I hadn't heard about functools yet, but I wonder what the benefit is over wrapping a function.

@de-vri-es do you think `functools` is necessary?

Reply:
From the official documentation, `partial` is roughly equivalent to
```
def partial(func, *args, **keywords):
    def newfunc(*fargs, **fkeywords):
        newkeywords = keywords.copy()
        newkeywords.update(fkeywords)
        return func(*(args + fargs), **newkeywords)
    newfunc.func = func
    newfunc.args = args
    newfunc.keywords = keywords
    return newfunc
```
I could add this wrapper somewhere if you prefer (you'll have to tell me what would be a good place for it, though). What I thought is that it's better to not reinvent the wheel, since `functools` is a standard library.

Reply:
I had the same doubt at first, but `functools` is part of the python standard library for both >2.5 and >3.0 so I don't see a problem with using it.

Reply:
Alright, sounds good! Maybe we should use it more often then ^^